Sources/Widgets/Widgets3D/AngleWidget/behavior.js

```diff
@@ -70,14 +71,14 @@ export default function widgetBehavior(publicAPI, model) {
     if (
       model.pickable &&
       model.dragable &&
-      model.manipulator &&
       model.activeState &&
       model.activeState.getActive() &&
       !ignoreKey(callData)
     ) {
-      model.manipulator.setOrigin(model.activeState.getOrigin());
-      model.manipulator.setNormal(model._camera.getDirectionOfProjection());
-      const worldCoords = model.manipulator.handleEvent(
+      const manipulator = model.activeState.getManipulator
+        ? model.activeState.getManipulator()
+        : model.manipulator;
```

Comment:
I could eventually do the following but I fear it can cause confusion.
```js
const manipulator = model.activeState.getManipulator 
    ? model.activeState.getManipulator() || model.manipulator 
    : model.manipulator;
```
I can't find a case where we would want to have a handle with `getManipulator()` returning null. This behavior would probably be an error and it would be best to notify the user rather than use an alternative manipulator. What do you think?

Reply:
`const manipulator = model.activeState.?getManipulator() || model.manipulator`

Interestingly enough, setting a null manipulator onto a handle would be a nice way to "disable" mouse events on such handle (and not for the whole widget).